docs/page.rst

```diff
@@ -986,17 +991,63 @@ In a nutshell, this is what you can do with PyMuPDF:
 
       .. note::
 
-         1. The method detects multiple insertions of the same image (like in above example) and will store its data only on the first execution. This is even true (although less performant), if using the default `xref=0`.
-         
-         2. The method cannot detect if the same image had already been part of the file before opening it.
-
-         3. You can use this method to provide a background or foreground image for the page, like a copyright or a watermark. Please remember, that watermarks require a transparent image if put in foreground ...
-
-         4. The image may be inserted uncompressed, e.g. if a *Pixmap* is used or if the image has an alpha channel. Therefore, consider using *deflate=True* when saving the file. In addition, there exist effective ways to control the image size -- even if transparency comes into play. Have a look at `this <https://pymupdf.readthedocs.io/en/latest/faq.html#how-to-add-images-to-a-pdf-page>`_ section of the documentation.
-
-         5. The image is stored in the PDF in its original quality. This may be much better than what you ever need for your display. Consider **decreasing the image size** before insertion -- e.g. by using the pixmap option and then shrinking it or scaling it down (see :ref:`Pixmap` chapter). The PIL method *Image.thumbnail()* can also be used for that purpose. The file size savings can be very significant.
-
-         6. Another efficient way to display the same image on multiple pages is another method: :meth:`show_pdf_page`. Consult :meth:`Document.convert_to_pdf` for how to obtain intermediary PDFs usable for that method. Demo script `fitz-logo.py <https://github.com/pymupdf/PyMuPDF-Utilities/tree/master/demo/fitz-logo.py>`_ implements a fairly complete approach.
+         1.
+           The method detects multiple insertions of the same image (like in
+           above example) and will store its data only on the first execution.
+           This is even true (although less performant), if using the default
+           `xref=0`.
+         2.
+           The method cannot detect if the same image had already been part of
+           the file before opening it.
+
+         3.
+           You can use this method to provide a background or foreground image
+           for the page, like a copyright or a watermark. Please remember, that
+           watermarks require a transparent image if put in foreground ...
+
+         4.
+           The image may be inserted uncompressed, e.g. if a *Pixmap* is used
+           or if the image has an alpha channel. Therefore, consider using
+           *deflate=True* when saving the file. In addition, there exist
```

Comment:
"there exist" sounds weird to me, I would just say "there are"

Reply:
Agreed, fixed in my tree.